crates/devices/virtio-vsock/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "virtio-vsock"
-version = "0.3.0"
+version = "0.4.0"
```

Comment:
Seem there's no functional/interface change, so should it be "0.4.0" or "0.3.1"?